src/lidar/read_rp_lidar.cpp

```diff
@@ -12,21 +13,14 @@
 
 using namespace navtypes;
 
-constexpr uint32_t RPLIDAR_BAUDRATE = 115200;
-const double MM_TO_M = 1000;
-
-std::atomic<bool> lidar_initialized(false);
-std::atomic<datatime_t> lidar_time(datatime_t{});
-RPLidar rp_lidar("/dev/ttyUSB0", RPLIDAR_BAUDRATE);
+bool lidar_initialized = false;
+RPLidar rp_lidar(Constants::Lidar::RP_PATH, Constants::Lidar::RPLIDAR_S1_BAUDRATE);
 std::thread lidar_thread;
 DataPoint<points_t> last_points = {};
 std::mutex points_lock;
 
 namespace lidar {
 
-/**
- * @brief RP Lidar continuously scans environment, updates current timeframe with lidar data
- */
 void readLidarLoop() {
     using namespace std::chrono;
```

Comment:
Is this still needed?

Reply:
For some reason the line isn't highlighted so by "this" I mean the `using namespace std::chrono` line